For Linux only: use the same background color for all selections.
diff -r -du src/Editor.cxx src/Editor.cxx
Index: src/Editor.cxx
===================================================================
--- src/Editor.cxx	(revision 73425)
+++ src/Editor.cxx	(working copy)
@@ -8456,7 +8456,7 @@
 
 	case SCI_SETSELBACK:
 		vs.selbackset = wParam != 0;
-		vs.selbackground = ColourDesired(lParam);
+		vs.selbackground = vs.selbackground2 = ColourDesired(lParam);
 		vs.selAdditionalBackground = ColourDesired(lParam);
 		InvalidateStyleRedraw();
 		break;
